sdk/resourcemanager/Azure.ResourceManager/tests/Azure.ResourceManager.Tests.csproj

```diff
@@ -31,7 +31,7 @@
   </ItemGroup>
   <ItemGroup>
     <Compile Include="$(TestFrameworkSupportFiles)" LinkBase="Shared\TestFramework" />
-    <Compile Include="..\..\..\..\common\ManagementTestShared\Redesign\*.cs" LinkBase="Shared\Mgmt" />
+    <Compile Include="..\..\..\..\common\ManagementTestShared\Temp\*.cs" LinkBase="TestShared" />
```

Comment:
The build target [here](https://github.com/Azure/azure-sdk-for-net/blob/d7998e345972682fadd443b0a23eab801e93721b/eng/Azure.Management.Test.targets#L12) already introduces the removed shared files into resourcemanager project, therefore we do not need to do it again.
I wonder why we previously did not get a compilation error since we have two copies for all those shared files.

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/41159 for all RPs